### ISSUE TITLE
Make optional warmup_end_value in create_lr_scheduler_with_warmup

### ISSUE THREAD

**Issue by vfdev-5** [pytorch/ignite#855]
## 🚀 Feature

It would be interesting to setup a lr scheduler without specifying `warmup_end_value` which can be deduced from provided optimizer:
```python
import torch
from torch.optim.lr_scheduler import ExponentialLR

t1 = torch.tensor([0.0])
opt = torch.optim.SGD([t1], lr=1.0)

torch_lr_scheduler = ExponentialLR(optimizer=opt, gamma=0.98)
lr_values = [None] * 100
scheduler = create_lr_scheduler_with_warmup(torch_lr_scheduler,
                                            warmup_start_value=0.0,
                                            warmup_end_value=None,  # warmup_end_value -> 1.0
                                            warmup_duration=10,
                                            output_simulated_values=lr_values)

print(lr_values)
```
